community.md

```diff
@@ -29,3 +29,16 @@ All Software developed for the OSCAR project is available on [GitHub](https://gi
 If you want to contribute to OSCAR, please fork the repository you want to contribute to
 and provide a pull request.
 
+Here is a list of all official repositories of the OSCAR project:
+
+* [AbstractAlgebra.jl](https://github.com/Nemocas/AbstractAlgebra.jl) - Generic abstract algebra and abstract type definitions
+* [GAP.jl](https://github.com/oscar-system/GAP.jl/) - GAP 4 to Julia bidirectional interface
```

Comment:
Could you also please add

[HomalgProject.jl](https://github.com/homalg-project/HomalgProject.jl) - The homalg project compatibility package for Julia

Reply:
Certainly!

Reply:
Thanks.